src/content/contributing/website/style_guide.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Docs Style Guide"
+date: 2020-04-29T19:09:55+02:00
+weight: 5
+---
+
+## Documentation Style Guide
+
+This guide is meant to help keep our documentation consistent and ease the
+contribution and review process.
+
+Submariner follows the [Kubernetes Documentation Style Guide][kube docs guide]
+wherever relevant. This is a Submariner-spicific extention of those practices.
+
+### Submariner.io word list
+
+A list of Submariner-specific terms and words to be used consistently across
+the site.
+
+Term | Usage
+:--- | :----
+Submariner | The project name *Submariner* should always be capitalized.
+Admral | The project name *Admral* should always be capitalized.
+Lighthouse | The project name *Lighthouse* should always be capitalized.
+Coastguard | The project name *Coastguard* should always be capitalized.
+Shipyard | The project name *Shipyard* should always be capitalized.
+`subctl` | The artifact `subctl` should not be capitalized and should be formatted in code style.
```

Comment:
Should we extend this to any script or command?

Reply:
Maybe, I'm not sure. I think this is partly covered by https://kubernetes.io/docs/contribute/style/style-guide/#use-code-style-for-kubernetes-command-tool-and-component-names, and I just intended to be explicit with major/common Submariner terms.